fedimint-logging/Cargo.toml

```diff
@@ -24,4 +24,4 @@ tracing-opentelemetry = { version = "0.23.0", optional = true}
 opentelemetry = { version = "0.22.0", optional = true }
 opentelemetry-jaeger = { version = "0.21.0", optional = true }
 console-subscriber = { version = "0.2.0", optional = true }
-tracing-chrome = { version = "0.7.1", optional = true}
+tracing-chrome = { version = "0.7.2", optional = true}
```

Comment:
We should reconsider if we need that dep, have we ever used it @maan2003?

Reply:
nope never used it